Makefile

```diff
@@ -8,6 +8,7 @@ BUCKET_NAME ?= cloudprowess
 FLAVOR ?= custom
 LOCAL_PATH := build/s3/$(FLAVOR)
 LICENSE_FILES := LICENSE
+TERRAFORM_VERSION := 0.11.3
```

Comment:
Although it's good to stick to one version, I don't know how to feel about all the upgrades of terraform, as they upgrade it at a fairly good rhythm.

Reply:
I guess this is just because we need to create the download URL.

I would prefer to run it from a container, using the latest available Terraform Docker image.